### PR TITLE
Update contract addresses

### PR DESCRIPTION
**Pull request by dospore**
Update contract addresses

### CODE REVIEW

File: src/data/poolList.ts [hunk 1]
```diff
@@ -234,120 +234,150 @@ export const poolList: Record<KnownNetwork, StaticPoolInfo[]> = {
...
+        {
+            // 3x
+            name: '3-ETH/USDC-SMA',
+            address: '0xAA388c11f738a77945FB96d86Db8D6F7dCc5CCd7',
+            leverage: 3,
+            updateInterval: FIVE_MINUTES,
+            frontRunningInterval: THIRTY_SECONDS,
+            keeper: '0x6198BEc9C4163fFc80f3CCCF85Ab2229D4f9548c',
+            committer: {
+                address: '0x96025f801eE19dfef560dbEf3122B254fD529550',
```
Comment (ScaredIbis) [inline src/data/poolList.ts:363]: this should be `0xa50013C89ECbACd873c44fc18264ebeA4DC41654`

File: src/data/poolList.ts [hunk 2]
```diff
@@ -234,120 +234,150 @@ export const poolList: Record<KnownNetwork, StaticPoolInfo[]> = {
         {
             // 1x
             name: '1-BTC/USDC',
-            address: '0x1b9A08Bd1b976fd91a355625509FbFcbbF34fb20',
+            address: '0x4eDc3814959705E59afeabbb52117950333c0721',
             leverage: 1,
             updateInterval: FIVE_MINUTES,
             frontRunningInterval: THIRTY_SECONDS,
```
Comment (ScaredIbis) [inline src/data/poolList.ts:240]: these are all one hour upkeep interval + 5 minute frontrunning interval for the current deploy

**Review by ScaredIbis — CHANGES_REQUESTED** [review]

File: src/data/poolList.ts [hunk 3]
```diff
@@ -234,122 +235,152 @@ export const poolList: Record<KnownNetwork, StaticPoolInfo[]> = {
...
             // 3x
             name: '3-ETH/USDC',
-            address: '0xe3cAD84c29775399a25c6Ff040FE141b4eD10047',
+            address: '0x11F7FD07422dB02AceE8009C002146a3b0980D92',
+            leverage: 3,
+            updateInterval: FIVE_MINUTES,
+            frontRunningInterval: THIRTY_SECONDS,
+            keeper: '0xFBCbB7184935122Fdf2c7e8b69a3207Ece722749',
+            committer: {
+                address: '0x0212e821a5383dB36362f197fdb0Dc93A75508C4',
```
Comment (ScaredIbis) [inline src/data/poolList.ts:319]: this should be `0x29EAAcB6FF4B883F1Ed630425cE89932601753E1` my bad, have updated notion doc to match (using graph as source of truth)

File: src/data/poolList.ts [hunk 4]
```diff
@@ -234,122 +235,152 @@ export const poolList: Record<KnownNetwork, StaticPoolInfo[]> = {
...
+                symbol: '3L-ETH/USD',
+                decimals: TEST_QUOTE_TOKEN.decimals,
+            },
+            shortToken: {
+                name: '3-SHORT-ETH/USD',
+                address: '0xCdaB5Df545A3650faeD921E7202e0bc3f02e8EC7',
+                symbol: '3S-ETH/USD',
+                decimals: TEST_QUOTE_TOKEN.decimals,
             },
+            quoteToken: TEST_QUOTE_TOKEN
```
Comment (ScaredIbis) [inline src/data/poolList.ts:383]: I think this is a duplicate entry for the 3ETH SMA pool

File: src/utils/constants.ts [hunk 5]
```diff
@@ -8,4 +8,11 @@ export const NETWORKS = {
 
 export const TCR_DECIMALS = 18;
 export const TEST_TOKEN_DECIMALS = 18;
-export const USDC_DECIMALS = 6;
\ No newline at end of file
+export const USDC_DECIMALS = 6;
+
+export const TEST_QUOTE_TOKEN = {
+    name: 'USDC',
+    symbol: 'USDC',
```
Comment (ScaredIbis) [inline src/utils/constants.ts:15]: don't think it matters and in fact probably will break icons but this is actually `TUSDC`

**Review by ScaredIbis — CHANGES_REQUESTED** [review]

File: src/data/poolList.ts (hunk 3 above)
Comment (dospore) [inline src/data/poolList.ts:319]:
```suggestion
                address: '0x29EAAcB6FF4B883F1Ed630425cE89932601753E1',
```

**Review by ScaredIbis — APPROVED** [review]

**Pull request merged**